python-bindings/overview_article/tasks.py

```diff
@@ -1,48 +1,95 @@
 """ Task definitions for invoke command line utility for python bindings
-    overview article. """
+    overview article.
+
+    If anything is causing trouble get in touch at
+    Znunu @ github or VimVim @ fcZBB2v @ discord"""
```

Comment:
I appreciate you taking ownership for your work, but I'd propose rephrasing this to:
"Special thanks to Znunu @ github for working through the Windows compatibility issues." 

(which, I know, would sound if you said it, but it will sound like it's coming from me when we merge it :) )

Reply:
It's not so important for me to be credited. Just wanted to add a quick and easy way for someone to get in touch. Someone might encounter a bug or have a problem, and thus open the file. Maybe I'm doing this the wrong way. I can remove it again.

Reply:
I've got a question in with Dan to see what he wants.  I'll let you know when he gets back to me (I've set a reminder that if I don't hear from him by Tuesday night, I'll ding him again.)